build.py

```diff
@@ -522,42 +525,39 @@ def build(version=None,
         logging.info("Building target: {}".format(target))
         build_command = ""
 
-        # Handle static binary output
-        if static is True or "static_" in arch:
-            if "static_" in arch:
-                static = True
-                arch = arch.replace("static_", "")
-            build_command += "CGO_ENABLED=0 "
+        build_command += "CGO_ENABLED=1 "
 
         # Handle variations in architecture output
         fullarch = arch
-        if arch == "i386" or arch == "i686":
-            arch = "386"
-        elif arch == "aarch64" or arch == "arm64":
+        if  arch == "aarch64" or arch == "arm64":
             arch = "arm64"
-        elif "arm" in arch:
-            arch = "arm"
-        build_command += "GOOS={} GOARCH={} ".format(platform, arch)
+
+        if platform == "linux":
+            if arch == "amd64":
+                tags += ["netgo", "osusergo", "static_build"]
+            if arch == "arm64":
+                cc = "aarch64-unknown-linux-musl-cc"
+                tags += ["netgo", "osusergo", "static_build", "noasm"]
+        elif platform == "darwin" and arch == "amd64":
+            cc = "x86_64-apple-darwin15-clang"
+            tags += [ "netgo", "osusergo"]
+        elif  platform == "windows" and arch == "amd64":
+            cc = "x86_64-w64-mingw32-gcc"
```

Comment:
While Influx doesn't officially support the windows build, it would be awesome if someone wanted to test it for me (I don't have a windows machine).